CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift

```diff
@@ -40,8 +58,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         CKContainer.default().add(badgeResetOperation)
     }
 
+    // MARK: Notifications
+    
     func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-
+    
```

Comment:
@barthoffman Here.

Reply:
Note that CloudKit does handle device tokens for you, so you don't need to implement the application(_:didRegisterForRemoteNotificationsWithDeviceToken:) method, unless you need that for another purpose.